public/docs/_examples/intro/dart/lib/logger_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.logger_service;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Injectable()
+class Logger {
```

Comment:
Tough one. I worry that adding in another package here would add complexity.

Might be worth mentioning it in a comment though?